po/vi.po

```diff
@@ -70,7 +70,7 @@ msgstr ""
 
 #: src/SUMMARY.md src/hello-world/playground.md
 msgid "Playground"
-msgstr ""
+msgstr "Nơi thử nghiệm (*Playground*)"
```

Comment:
I suggest using "Sân chơi" as a translation of "Playground". "Nơi thử nghiệm" sounds too gloomy for me (more like a test site than a playground"